src/ol/layer/Tile.js

```diff
@@ -53,9 +53,7 @@ inherits(TileLayer, Layer);
  * @api
  */
 TileLayer.prototype.getPreload = function() {
-  return (
-  /** @type {number} */ this.get(_ol_layer_TileProperty_.PRELOAD)
-  );
+  return (/** @type {number} */ this.get(TileProperty.PRELOAD));
```

Comment:
These parens might have gotten messed up by the initial code transform.  Shouldn't this be like:
```js
return /** @type {number} */ (this.get(TileProperty.PRELOAD));
```

(I'll be happy when we get rid of these typecasts altogether.)

Reply:
in a8f4348